lte/cloud/go/services/subscriberdb/internal_servicer/lookup.go

```diff
@@ -0,0 +1,170 @@
+/*
```

Comment:
lookup.go is recognised as a new file here. Please use mv command to move existing files instead of deleting and adding a new file with same contents._